### PR TITLE
PR for code review - Capstone project

### PR DESCRIPTION
**Pull request by wickathou**
This is Analytics Reporting, an app that takes live data from an Airtable database which collects user traffic for sites that I manage.

The app uses the Airtable API to fetch data and Redux to keep track of the state of the app.

Features:
- Vite + Tailwind
- Jest test for components
- API calls

[Video Walkthrough](https://www.loom.com/share/4c6abe5dfa1645a5bee7830e8b149a6a)

[Live view](https://wickathou.github.io/analytics-reporting)

[Most interesting part of the project](https://www.loom.com/share/e78dac7adb5446919af71b0741e67c6f)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,96 @@
...
+
+Feel free to check the [issues page](https://github.com/wickathou/analytics-reporting/issues).
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project feel free to fork it or star it
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks Microverse, learn how to code > [Join Microverse](https://www.microverse.org/?grsf=9m3hq6)
```
Comment (jerryowusu) [inline README.md]: Kindly ensure that your readme file contains and "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

File: src/test/Components.test.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+describe('DetailList tests', () => {
+  test('to render DetailList: ', () => {
+    render(
+      <BrowserRouter>
+        <DetailList page={mockedItem} />
+      </BrowserRouter>,
+    );
+    expect(screen).toMatchSnapshot();
+  });
+});
```
Comment (jerryowusu) [inline src/test/Components.test.js]: Nice work adding test for the components. However, as part of the project requirements, kindly add tests for some of the pure functions of the redux code. :+1:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @wickathou , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

almost there


## Highlights
Great effort so far :heavy_check_mark: 
No linter errors :heavy_check_mark: 

## Required Changes ♻️

- [ ] The link to the License file is broken. Kindly check and make sure it works fine to give your project a more proffessional appeal. :+1: 
- [ ] As part of the project requirements, kindly  highlight some interesting piece of code or something you built that you are very proud of in the project documentation video. (loom video) :wink: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @wickathou, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.


 ## Highlights
✅ Linters are passing

 ## Required Changes ♻️
_Check the comments under the review._
I had to add comment a here because it could not be done inline

**[REQUIRED] As requested by the previous reviewer kindly point out a piece of code you find interesting. This is a requirement for this project.**
 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by wickathou** [comment]
> Hi @wickathou,
> 
> Great work on making the changes requested by a previous reviewer clap You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet.
> 
> ## Highlights
> white_check_mark Linters are passing
> 
> ## Required Changes 
> _Check the comments under the review._ I had to add comment a here because it could not be done inline
> 
> **[REQUIRED] As requested by the previous reviewer kindly point out a piece of code you find interesting. This is a requirement for this project.**
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Done https://www.loom.com/share/e78dac7adb5446919af71b0741e67c6f

Thank you

**Review by victorokoroji — APPROVED** [review]
Hi @wickathou, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**